modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -11,37 +13,51 @@ This guide contains an overview of the changes in Canvas Kit v11. Please
 [reach out](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you have
 any questions.
 
+## Why you should migrate
+
+Canvas Kit v11 is a transition into a new
+[styling future](https://github.com/Workday/canvas-kit/discussions/2265). Theming and building an in
+sync Canvas Kit CSS has been at the top of our minds. We've started using our new
+[Canvas Tokens Web](https://workday.github.io/canvas-tokens/?path=/docs/docs-getting-started--docs)
+package to take advantage of
+[CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) and
+provide semantic tokens that can translate to theming components.
+
+A note to the reader:
+
+- While we still support our old tokens from `@workday/canvas-kit-react/tokens` we encourage
+  consumers to start using our new tokens from `@workday/canvas-tokens-web`. You can find more info
+  in our [docs](https://workday.github.io/canvas-tokens/?path=/docs/docs-getting-started--docs).
+- In this release, we've introduced a
+  [new styling API](https://workday.github.io/canvas-kit/?path=/docs/styling-basics--create-stencil).
+  This shouldn't effect the way you currently style your components. Because we're moving away from
+  Emotion, support for style props will eventually be removed, however, this API provide backwards
+  compatability. If you want to slowly move off of Emotion as well, you can style styling components
+  via the `cs` prop or our new styling utilities.
+
+## Table of contents
+
 - [Codemod](#codemod)
-- [Component Promotions](#component-promotions)
```

Comment:
Do we not want to call this out since this is a whole new component and not just a change?

Reply:
In previous versions we've put this under `Component Updates` so I kept that way. Notice table is under component updates, and that's where the promotion is called out a long with any other updates to it!

Reply:
Perfect. I think that's a good way to go about it